cmd/kubeadm/app/phases/controlplane/volumes.go

```diff
@@ -42,14 +42,16 @@ var caCertsPkiVolumePath = "/etc/pki"
 
 // getHostPathVolumesForTheControlPlane gets the required hostPath volumes and mounts for the control plane
 func getHostPathVolumesForTheControlPlane(cfg *kubeadmapi.MasterConfiguration) controlPlaneHostPathMounts {
+	hostPathDirectoryOrCreate := v1.HostPathDirectoryOrCreate
```

Comment:
nit: why declare temporary variables?

Reply:
The `HostPathType` is supposed to be a pointer. We have to declare such temporary variables.